### PR TITLE
Creating a new util folder for testing

### LINKED ISSUE #497: Move util/testing-utils to new test/util directory
This keeps all test related code in the `test/` directory, you should also move `test/mocks/` into this new `test/util`.

The directory should look like:

```
├── test/util
│   ├── mocks
│   │   ├── Date.js
│   │   ├── TokenValidFunctions.js
│   │   └── react-router-dom.js
│   ├── tools
│   │   └── tools.js
```

### PR DESCRIPTION
**Pull request by law-nguyen**
Resolves #497

### CODE REVIEW

**Comment by ThaiQ** [comment]
frontendddd

File: test/util/tools/SceApiTester.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      .send(params)
+      .then(function(res) {
+        response = res;
+      })
+      .catch(err =>{
+        throw err;
+      });
+    return response;
+  }
+  async sendPostRequestToken(token, endpoint, params){
```
Comment (evanugarte) [inline test/util/tools/SceApiTester.js:27]:
```suggestion
  async sendPostRequestWithToken(token, endpoint, params){
```

File: test/util/tools/SceApiTester.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      .send(params)
+      .then(function(res) {
+        response = res;
+      })
+      .catch(err =>{
+        throw err;
+      });
+    return response;
+  }
+  async sendPostRequestWithToken(token, endpoint, params){
```
Comment (evanugarte) [inline test/util/tools/SceApiTester.js:27]: add space between this function and end of the one above
```suggestion

  async sendPostRequestWithToken(token, endpoint, params){
```

File: test/util/tools/SceApiTester.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    let response = null;
+    await chai
+      .request(this.app)
+      .post(endpoint)
+      .send({token, ...params})
```
Comment (evanugarte) [inline test/util/tools/SceApiTester.js:32]:
```suggestion
      .send({ token, ...params })
```

File: test/util/tools/SceApiTester.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+const chaiHttp = require('chai-http');
+chai.should();
+chai.use(chaiHttp);
+
+class SceApiTester{
+  constructor(app){
+    this.app = app;
+  }
+
+  async sendPostRequest(endpoint, params){
```
Comment (evanugarte) [inline test/util/tools/SceApiTester.js:13]: consider adding documentation to this class, e.g.
1. [`api/util/PathParser.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/util/PathParser.js)
1. [`api/SceHttpServer.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/SceHttpServer.js)
1. [`api/mailer/SceGoogleApiHandler.js`](https://github.com/SCE-Development/Core-v4/blob/dev/api/mailer/SceGoogleApiHandler.js)

**Pull request merged**